helm/ffc-sfd-business/values.yaml

```diff
@@ -14,12 +14,12 @@ deployment: {}
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 container:
   memCpuTier: L
-  port: 3000
+  port: 3003
 
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 livenessProbe:
   path: /healthz
```

Comment:
does this need to be prefixed with `/business`?

Reply:
Yep